src/plugins/janus_videoroom.c

```diff
@@ -2433,6 +2433,8 @@ static void janus_videoroom_publisher_dereference_nodebug(janus_videoroom_publis
 	janus_refcount_decrease_nodebug(&p->ref);
 }
 
+static void janus_videoroom_session_destroy(janus_videoroom_session *session);
```

Comment:
Not sure if you want the forward declaration or if one of the two functions should be moved.